packages/test-utils/src/wrapper.js

```diff
@@ -208,7 +208,7 @@ export default class Wrapper implements BaseWrapper {
   }
 
   /**
-   * Utility to check wrapper exists. Returns true as Wrapper always exists
+   * Utility to check wrapper exists.
```

Comment:
This appeared to be an outdated comment for the `exists` method, since it a Wrapper doesn't always exist, since it can return `false` :slightly_smiling_face:

Reply:
indeed!